src/java/org/apache/cassandra/sensors/SensorsRegistry.java

```diff
@@ -123,77 +128,6 @@ public Optional<Sensor> getSensor(Context context, Type type)
         return Optional.ofNullable(getSensorFast(context, type));
     }
 
-    /**
-     * To get best perfromance we are not returning Optional here
-     */
-    @Nullable
-    public Sensor getSensorFast(Context context, Type type)
-    {
-        Sensor[] typeSensors = identity.get(context);
-        return  typeSensors != null ? typeSensors[type.ordinal()] : null;
-    }
-
-    /**
-     * To get best perfromance we are not returning Optional here
-     */
-    @Nullable
-    public Sensor getOrCreateSensorFast(Context context, Type type)
-    {
-        Sensor sensor = getSensorFast(context, type);
-        if (sensor != null)
-            return sensor;
-
-        stripedUpdateLock.getAt(getLockStripe(context.getKeyspace().hashCode())).readLock().lock();
-        try
-        {
-            if (!keyspaces.contains(context.getKeyspace()) || !tableIds.contains(context.getTableId()))
-                return null;
-
-            AtomicReference<Sensor> newSensor = new AtomicReference<>();
-            Sensor[] typeSensors = identity.computeIfAbsent(context, key -> {
-                Sensor[] newTypeSensors = new Sensor[Type.values().length];
-                newTypeSensors[type.ordinal()] = createNewSensor(context, type);
-                newSensor.set(newTypeSensors[type.ordinal()]);
-                return newTypeSensors;
-            });
-            if (typeSensors == null)
-                return newSensor.get();
-
-            synchronized (typeSensorsLock)
```

Comment:
This was a bit of a bottleneck, also not a fan of nested locks ... replaced with `compute()` below.